doc/user/data/explain_plan_operators.yml

```diff
@@ -0,0 +1,223 @@
+operators:
+  - operator: Constant
+    plan_types: "optimized,raw"
+    description: |
+      Always produces the same collection of rows.
+    uses_memory: False
+    memory_details: ""
+    expansive: False
+
+    example: |
+      ```mzsql
+      Constant
+      - ((1, 2) x 2)
+      - (3, 4)
+      ```
+
+  - operator: Get
+    plan_types: "optimized,raw"
+    description: |
+      Produces rows from either an existing source/view or from a previous
+      operator in the same plan.
+    uses_memory: False
+    memory_details: ""
+    expansive: False
+
+    example: "`Get materialize.public.ordered`"
+
+  - operator: Project
+    plan_types: "optimized,raw"
+    description: |
```

Comment:
Do we already explain column numbering somewhere? Might be good to link it here.

Reply:
We have a little paragraph. Will add the link.  Thank you!

Reply:
Just to manage expectations, the paragraph could be better ... but, that's on the list of todos 😄